VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/services/LoginService.java

```diff
@@ -0,0 +1,26 @@
+package edu.vt.cs.vtcare.vtcareservice.services;
+
+import edu.vt.cs.vtcare.vtcareservice.dao.LoginDao;
+import java.sql.SQLException;
+
+public class LoginService {
+    private final LoginDao loginDao;
```

Comment:
Here use appropriate provider or patients dao. As mentioned in a previous comment, please add the SQLs for provider and patients in their appropriate dao classes and delete the loginDao altogether.